src/sentry/static/sentry/app/views/groupDetails/actions.jsx

```diff
@@ -18,26 +20,24 @@ import TooltipMixin from '../../mixins/tooltip';
 import ResolveActions from '../../components/actions/resolve';
 import IgnoreActions from '../../components/actions/ignore';
 
-const DeleteActions = React.createClass({
-  propTypes: {
+class DeleteActions extends React.Component {
+  static propTypes = {
     organization: PropTypes.object.isRequired,
     project: PropTypes.object.isRequired,
     onDelete: PropTypes.func.isRequired,
     onDiscard: PropTypes.func.isRequired,
-  },
+  };
 
-  getInitialState() {
-    return {
-      hooksDisabled: HookStore.get('project:discard-groups:disabled'),
-    };
-  },
+  state = {
```

Comment:
Let's make sure we initialize state in constructor to be consistent

Reply:
It's on the ~4 components that are now es6 components. Unfortunately I think we have to do it manually.